ChangeLog

```diff
@@ -40,6 +40,7 @@ ver. 0.9.2 (2014/XX/XXX) - wanna-be-released
    * firewallcmd-* actions: split output into separate lines for grepping (gh-908)
    * Guard unicode encode/decode issues while storing records in the database.
      Fixes "binding parameter error (unsupported type)" (gh-973), thanks to kot for reporting
+   * filter.d/asterisk.conf - Match hacking attempt IP instead of asterisk server IP
```

Comment:
The removed failregex logs the asterisk server's IP address versus the new failregex which logs the remote client/bad guy/IP we want fail2ban to ban.

I wasn't sure of a concise way to reference the bad guy's IP in the scenario, so I used "hacking attempt IP" - granted it's not the best phrase.

I'd say the dropped failregex incorrectly bans the server fail2ban is protecting, in this case the asterisk server itself. I'm not sure 'originating server' is especially clear either.